src/io/flutter/sdk/FlutterSdkUtil.java

```diff
@@ -35,13 +36,25 @@
 
 
 public class FlutterSdkUtil {
+  /** The environment variable to use to tell the flutter tool which app is driving it. */
+  public static final String FLUTTER_ENVIRONMENT = "FLUTTER_ENVIRONMENT";
```

Comment:
The name seems ambiguous - "environment" could mean anything. For the analysis server we use "client_id" and "client_version". Should we do the same here?